worker/include/RTC/RTCP/CompoundPacket.h

```diff
@@ -12,21 +12,17 @@ namespace RTC { namespace RTCP
 	class CompoundPacket
 	{
 	public:
-		~CompoundPacket();
 
-		uint8_t* GetRaw();
 		size_t GetSize();
```

Comment:
Don't worry, we can do a consistent renaming later. It's just that I don't yet understand the difference between `SdesItem.GetSize` and `SdesItem.GetLength`:

```c++
inline
size_t SdesItem::GetSize()
{
	return 2 + size_t(this->header->length);
}

inline
uint8_t SdesItem::GetLength()
{
	return this->header->length;
}
```